Js/main.js

```diff
@@ -0,0 +1,13 @@
+const data =require('./data.js')
+// const searchInput=document.querySelector('.search-box')
+//Search By Name
+// searchInput.addEventListener('keypress', searchByName(searchInput.value))
+const searchByName =((search)=>{
```

Comment:
it's better to  replace 'search' parameter with 'searchedName' to be more conventional